@here/harp-datasource-protocol/lib/DecodedTile.ts

```diff
@@ -59,6 +60,12 @@ export interface PathGeometry {
     path: Vector3Like[];
 }
 
+/**
+ * Attributes corresponding to some decoded geometry. It may be either a map
+ * of multiple attributes or just a number with the geometry's feature id.
```

Comment:
TODO (follow-up PRs): Assign directly feature id to the attribute map if we don't decode other attributes.